### DISCUSSION TITLE
Force installed extension update not happening

### DISCUSSION THREAD

**Discussion by sc231997** [microsoft/MicrosoftEdge-Extensions#141] [Q&A]
On Windows I am trying to Force install a extension by setting the value in the registry. The installation is happening but when I bump up the version the updated extension is not getting installed.
I have tried force update by clicking on Update in `edge://extension` page.
```
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\ExtensionInstallForcelist\1=1231432542543656;C:\edge.xml
```
content of `edge.xml`
```xml
<?xml version='1.0' encoding='utf-8'?>
<gupdate xmlns='http://www.google.com/update2/response' protocol='2.0'>
    <app appid='1231432542543656'>
        <updatecheck codebase='C:\my-extension.crx' version='3.4.1' />
    </app>
</gupdate>
```
I have mocked the extension-id above.

**Comment by ManikanthMSFT** [comment]
Hi @sc231997, sorry for the inconvenience caused. Could you please help us with your seller id and the extension name if the issue still persists?

**Reply by sc231997** [reply]
Hi @ManikanthMSFT 
I have not published my extension anywhere. We are trying to build an extension for our internal teams.

**Reply by ManikanthMSFT** [reply]
Hey @sc231997, could you please confirm have you created a partner center account to publish your extension?

**Reply by sc231997** [reply]
Nope i have not created that account
